src/NuGet.Core/NuGet.Packaging/Signing/Verification/SignatureTrustAndValidityVerificationProvider.cs

```diff
@@ -219,28 +219,28 @@ public SignatureTrustAndValidityVerificationProvider(IEnumerable<KeyValuePair<st
             Timestamp timestamp;
             var timestampSummary = GetTimestamp(signature, verifierSettings, out timestamp);
 
+            var status = signature.Verify(
+            timestamp,
```

Comment:
Indent here for better readability?

Reply:
Also can you put explicit type? Otherwise hard to follow what kind of value is here?

Reply:
If you are okay, I will create a follow up PR addressing above comments because this PR has passed all checks.